handlePanic.go

```diff
@@ -0,0 +1,121 @@
+package gramework
+
+import "strings"
+
+var errGotPanic = struct {
+	Code    int    `json:"code" xml:"code" csv:"code"`
+	Message string `json:"message" xml:"message" csv:"message"`
+}{
+	Code:    500,
+	Message: "Internal Server Error",
+}
+
+// DefaultPanicHandler serves error page or error response depending on ctx.ContentType()
+func DefaultPanicHandler(ctx *Context, panicReason interface{}) {
+	ctx.SetStatusCode(500)
+	if strings.HasPrefix(string(ctx.Request.Header.Peek("Accept")), "text/html") || strings.Contains(ctx.ContentType(), "text/html") {
+		ctx.HTML().WriteString(handledPanic)
```

Comment:
Error return value of `(*github.com/gramework/gramework/vendor/github.com/valyala/fasthttp.RequestCtx).WriteString` is not checked